headers/types/ground_mode/live_actor.h

```diff
@@ -0,0 +1,98 @@
+#ifndef HEADERS_TYPES_GROUND_MODE_LIVE_ACTOR_H_
+#define HEADERS_TYPES_GROUND_MODE_LIVE_ACTOR_H_
+
+// represent an actor present in the scene in the overworld (both during cutscenes and free-roams)
+struct live_actor {
+    struct monster_id_16 id; // The id of the Actor in the actor list
+    uint16_t kind;           // The species of the Actor
```

Comment:
Is this right? I don't actually know what these two fields represent other than what your comments say, so please correct me if I'm wrong.

`struct monster_id_16` corresponds to the Pokémon species, e.g., Bulbasaur, Ivysaur, etc. From the descriptions it seems like this should be:

```c
uint16_t id;               // The id of the Actor in the actor list
struct monster_id_16 kind; // The species of the actor
```